static/app/components/issues/groupList.tsx

```diff
@@ -125,6 +126,18 @@ class GroupList extends React.Component<Props, State> {
 
     const endpoint = this.getGroupListEndpoint();
 
+    const query = (queryParams ?? this.getQueryParams()).query;
+
+    // Check if the alert rule query
+    if (/ or | and /i.test(query)) {
```

Comment:
```suggestion
    if (/\s(or|and)\s/i.test(query)) {
```
`\s` is a bit more specific

Reply:
we might want to use our query parser for this